spec/session_spec.coffee

```diff
@@ -579,3 +579,13 @@ describe 'Session', ->
               @instance.then =>
                   expect(@biskoto.get(@basic_sa_cookie_name).version).to.equal @settings.cookies.version
                   done()
+
+      context 'when the shop_code is not present', ->
+        beforeEach ->
+          @init = =>
+            sa('session', 'connect', '')
+            @instance = new @session(@plugins_manager).run()
+
+        it 'does not call XDomainEngine', ->
```

Comment:
Can we check also that plugins_manager.notify is not called?

Reply:
agreed, an extra check wouldn't hurt